src/Components/Sidebar/Sidebar.tsx

```diff
@@ -7,9 +7,11 @@ import { HiOutlineUsers } from "react-icons/hi2";
 import { IoIosUnlock } from "react-icons/io";
 import { LiaHomeSolid } from "react-icons/lia";
 import { Menu, MenuItem, Sidebar } from 'react-pro-sidebar';
-import { Link, useLocation } from 'react-router-dom';
 import { Modal } from 'react-bootstrap';
+import { Link, useLocation } from 'react-router-dom';
 import { ChangePassword } from '@/Pages';
+import { FaProjectDiagram } from "react-icons/fa";
```

Comment:
You can rename the icons here and  give them a better name such as:
`import { FaProjectDiagram as ProjectsIcon } from "react-icons/fa";`